resources/resincli.robot

```diff
@@ -69,6 +69,28 @@ Device ${device_uuid} log should contain ${value}
     Process ${result}
     Should Contain    ${result.stdout}    ${value}
 
+Add ENV variable Hello with value World to application ${application_name} 
+    ${result} =  Run Process    resin   env     add     Hello    World    -a  ${application_name}
```

Comment:
I should have been clearer - I wanted a variable in the resources/resincli.robot and a fixed value in the test - That way this could be used to write other components.